src/components/ShoppingCart/ListRow.vue

```diff
@@ -153,6 +164,10 @@ export default {
       type: Number,
       default: 0,
     },
+    productPrice: {
+      type: Number,
+      default: 0,
+    },
```

Comment:
I think `productPrice` is already defined in `nft-or-collection` mixin and no need to be defined as prop